### PR TITLE
Add initial API concepts

### PR DESCRIPTION
**Pull request by Nashaath**
## Purpose
Add initial API concepts.

### CODE REVIEW

File: en/docs/apis/api-concepts.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+API Key Manager handles all security and key-related operations. 
+
+### API Gateway
+
+API Gateway acts as the entry point for an API request. It secures, protects, manages, and scales API calls. 
+
+
+## API Lifecycle
+
+API lifecycle comprises of the stages an API goes through from creation to retirement. API lifecycle is independent of its backend service. The lifecycle states are `Created`, `Prototyped`, `Published`, `Deprecated`, and `Retired`.
```
Comment (anupama-pathirage) [inline en/docs/apis/api-concepts.md:33]: The state `Blocked` is missing.

File: en/docs/apis/api-concepts.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Concepts
+
+Before you start managing APIs via Choreo, there are basic concepts that you need to understand. This section walks you through all the concepts you need to know. 
+ 
+## API
+
+An API in Choreo allows you to expose business functionality as a service to customers, partners, suppliers, and other stakeholders.
+
```
Comment (anupama-pathirage) [inline en/docs/apis/api-concepts.md:8]: Remove extra new lines. Check all the places

Comment (Nashaath) [reply en/docs/apis/api-concepts.md:6]: Removed extra new lines.

**Review by RukshiW — APPROVED** [review]

**Comment by anupama-pathirage** [comment]
@Nashaath Shouldn't we send this PR to dev  branch?

**Review by anupama-pathirage — APPROVED** [review]

**Pull request merged**